### DISCUSSION TITLE
Running Shell Script with SkyWalking Java Agent

### DISCUSSION THREAD

**Discussion by vishalkadam18** [apache/skywalking#12567] [Q&A]
Dear Team,

I’m interested in running a shell script (.sh) with the SkyWalking java agent. Can you confirm if this is possible and provide guidance on how to properly configure the script for the agent?

but i  am unable to run .sh file with skywalking java agent

Thanks

**Comment by wu-sheng** [comment]
What are you asking about? The agent is just a thing included in the Java bootstrap command. Nothing more.

If you want to delegate java command, that is your thing.
